core/commands/social/commend.js

```diff
@@ -52,20 +66,19 @@ const init = async function (msg, args) {
 
     P.target = Target.nick || (Target.user || Target).username;
     P.author = msg.member.nick || msg.author.username;
-    P.cmcount = (targetData.totalIn + 1) || 0;
+    P.cmcount = targetData.totalIn + 1 || 0;
     P.pplcount = targetData.whoIn.length + 1;
 
     const embed = new Embed()
       .thumbnail(`${paths.DASH}/build/rank.png`)
       .color("#3b9ea5")
-      .timestamp(new Date())
-      .description(`
+      .timestamp(new Date()).description(`
             ${$t("responses.commend.give", P)}
             ${$t("responses.commend.totals", P)}
             `);
 
     Progression.emit("command.commend.commit", { msg, userID: msg.author.id });
-    msg.channel.send({ embed });
+    await msg.channel.send({ embed });
```

Comment:
In my experience, not waiting for the promise to execute causes flaky behaviour.

Reply:
fair enough. personally I think awaiting every promise kind of defeats the purpose of async. Flaky behaviour could be addressed with proper `.then` and `.catch` which in this case wouldn't do much. 

Since this is not a particularly slow operation I guess we can afford to have `await` here for the sake of predictability